autoblocks/_impl/tracer.py

```diff
@@ -1,26 +1,48 @@
 import asyncio
+import contextvars
+import dataclasses
+import inspect
 import logging
 import uuid
 from contextlib import contextmanager
-from dataclasses import dataclass
 from datetime import datetime
 from datetime import timedelta
 from datetime import timezone
+from typing import Any
 from typing import Dict
+from typing import List
 from typing import Optional
 
 from autoblocks._impl import global_state
 from autoblocks._impl.config.constants import INGESTION_ENDPOINT
+from autoblocks._impl.testing.models import BaseEventEvaluator
+from autoblocks._impl.testing.models import Evaluation
+from autoblocks._impl.testing.models import TracerEvent
 from autoblocks._impl.util import AutoblocksEnvVar
-
-log = logging.getLogger(__name__)
+from autoblocks._impl.util import gather_with_max_concurrency
 
 
-@dataclass
+@dataclasses.dataclass()
 class SendEventResponse:
     trace_id: Optional[str]
 
 
+class ClickHouseEvaluationObject(Evaluation):
+    @classmethod
+    def to_json(cls, event_evaluation: Evaluation, evaluator_external_id: str) -> Dict[str, Any]:
```

Comment:
I think a class method is appropriate here - I will never build an instance of this class directly. It's sole purpose is to combine this data and spit out valid json so I can use it ultimately in the events `evaluations` array

Reply:
then i feel we don't need a class for this at all, can't we just build the json directly?